### DISCUSSION TITLE
Warning: Dipole moment not defined absolutely

### DISCUSSION THREAD

**Discussion by achyuttt** [dftbplus/dftbplus#1501] [General]
Hello everyone,
I noticed lately that my every output file has a warning:

Warning: dipole moment is not defined absolutely!

What does this mean?
Is this something that should be taken into account? 
What should I do to eliminate this error?

**Comment by aradi** [comment]
Periodic systems do not have well defined (origin independent) dipole moments, so the calculated value should be used with caution. This is what the warning is about...
